docs/framework/configure-apps/file-schema/remove-element-for-configsections.md

```diff
@@ -10,7 +10,7 @@ ms.assetid: ae4d82e0-e8fe-468c-81ab-46d63c4d66a8
 ---
 # \<remove> element for \<configSections>
 
-Removes a predefined section or section group.
+No impact on section or section group.
```

Comment:
If they don't do *anything*, then I think we should just remove them altogether.

Reply:
That's probably better, but review #3708, alot of docs refer to them and lots of SO cases. If we delete them, what do we redirect to ? The parent?

Reply:
Let's add a new section to this page https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/configuration-sections-schema and we can redirect them to it. It can be called "Deprecated elements" or something.